Libraries/Animated/src/AnimatedImplementation.js

```
@@ -1715,9 +1715,11 @@ function createAnimatedComponent(Component: any): any {
       var callback = () => {
         if (this._component.setNativeProps) {
           if (!this._propsAnimated.__isNative) {
-            this._component.setNativeProps(
-              this._propsAnimated.__getAnimatedValue()
-            );
+            if (this._component.viewConfig != null) {
```

Can you throw an error here instead of just dropping the change?